tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1804,6 +1859,25 @@ schedule(void)
             i->misc_metadata = loop_md;
           }
         }
+        if( BIH_VECTORIZE_WIDTH_FACTOR( bih ) )
```

Comment:
Nit: Existing coding style has braces on the same line as if. Also there are no additional spaces inside the condition after `(`.

Reply:
fixed